pkg/apihelper/k8shelpers.go

```diff
@@ -50,6 +51,7 @@ func (h K8sHelpers) GetTopologyClient() (*topologyclientset.Clientset, error) {
 	return topologyClient, nil
 }
 
+// GetNode retrieves the node object from the clientset
```

Comment:
nit: clientset is sort of irrelevant here as it's just an interface to the k8s api.

```suggestion
// GetNode retrieves one node object
```

Reply:
done